crates/fluvio-socket/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "fluvio-socket"
-version = "0.12.0"
+version = "0.13.0"
```

Comment:
Is there API incompatibility? If we are just adding a new API, then just need to bump up patch version

Reply:
I thought that implementing `Future`  on `AsyncResponse` potentially could be a breaking change, but `orphan rules` would not allow it, so you're right. Changed to 0.12.1.